module/zfs/metaslab.c

```diff
@@ -2616,6 +2619,9 @@ metaslab_init(metaslab_group_t *mg, uint64_t id, uint64_t object,
 	ms->ms_allocator = -1;
 	ms->ms_new = B_TRUE;
 
+	if (vd->vdev_ops == &vdev_draid_ops)
+		vdev_draid_metaslab_init(vd, &ms->ms_start, &ms->ms_size);
```

Comment:
all those new vdev ops, but not one for this?

Reply:
I thought about it, but decided it wasn't needed at the time.  Upon further reflection I think it's worthwhile to do it now so I went ahead and made the change.